workflow-templates/im-deploy-iis-website.yml

```diff
@@ -425,7 +417,7 @@ jobs:
         if: always() && env.ENVIRONMENT == 'prod' && steps.conclusion.outputs.workflow_conclusion == 'success'
         uses: im-open/post-status-to-teams-action@v1.1.4
         with:
-          title: ${{ needs.set-vars.outputs.WEBSITE_NAME }} Deployment to ${{ needs.set-vars.outputs.IIS_SERVER }} # TODO Verify this title gives enough info since it will be posted to the Prod Notifications channel
+          title: ${{ env.WEBSITE_NAME }} Deployment to ${{ env.IIS_SERVER }} # TODO Verify this title gives enough info since it will be posted to the Prod Notifications channel
```

Comment:
Since you've moved these variables into the deploy-site job you'll need to export them there so they are available as env variables here.

Reply:
Same as above.

Reply:
Same as above. :)